toonz/sources/toonzqt/styleeditor.cpp

```diff
@@ -1505,46 +1505,49 @@ PlainColorPage::PlainColorPage(QWidget *parent)
                        SLOT(onControlChanged(const ColorModel &, bool)));
   }
 
-  QPushButton *wheelShowButton = new QPushButton(tr("Wheel"), this);
-  QPushButton *hsvShowButton   = new QPushButton(tr("HSV"), this);
-  QPushButton *alphaShowButton = new QPushButton(tr("Alpha"), this);
-  QPushButton *rgbShowButton   = new QPushButton(tr("RGB"), this);
-
-  QFrame *wheelFrame = new QFrame(this);
+  m_wheelShowButton         = new QPushButton(tr("Wheel"), this);
+  m_hsvShowButton           = new QPushButton(tr("HSV"), this);
+  m_alphaShowButton         = new QPushButton(tr("Alpha"), this);
+  m_rgbShowButton           = new QPushButton(tr("RGB"), this);
+  m_toggleOrientationButton = new QPushButton(tr("↔"), this);
```

Comment:
In my environment with MSVC 2013 Japanese version, build failed because of the characters "↔" and "↕".
Can you please replace `"↔"` in lines 1512 and 1745 by `QChar(0x2194)`, and `"↕"` in line 1752 by `QChar(0x2195)` ? They will specify the same character with unicode.